virttest/qemu_vm.py

```diff
@@ -1465,23 +1483,13 @@ def sort_key(dev):
 
         vga = params.get("vga")
         if vga:
-            if vga != 'none':
-                devices.insert(StrDev('VGA-%s' % vga,
-                                      cmdline=add_vga(vga),
-                                      parent_bus={'aobject': 'pci.0'}))
-                if vga == 'qxl':
-                    qxl_dev_nr = int(params.get("qxl_dev_nr", 1))
-                    if qxl_dev_nr > 1:
-                        addr = int(params.get("qxl_base_addr", 29))
-                        cmdline = add_qxl(qxl_dev_nr, addr)
-                        devices.insert(StrDev('qxl', cmdline=cmdline))
-            else:
-                devices.insert(StrDev('VGA-none', cmdline=add_vga(vga)))
-
-        elif params.get('defaults', 'no') != 'no':  # by default add cirrus
```

Comment:
it seems no default vga any more

Reply:
Yes, because `vga = std` always be set in `base.cfg`, I think it is okay to remove the above logic.

Reply:
Well I'd suggest keeping the default as people might be using custom cart config files without it defined.